packages/qwik/src/core/shared/jsx/types/jsx-qwik-events.ts

```diff
@@ -16,6 +16,8 @@ export type QwikSymbolEvent = CustomEvent<{
 export type QwikInitEvent = CustomEvent<{}>;
 /** Emitted by qwik-loader on document when the document first becomes idle @public */
 export type QwikIdleEvent = CustomEvent<{}>;
+/** Emitted by qwik-core on document when the a view transition start @public */
+export type QwikViewTransitionEvent = CustomEvent<ViewTransition>;
```

Comment:
and here?

Reply:
I'm going to leave these in, the #7391 pr is missing them and I'm worried it will be forgotten